application/src/main/java/org/togetherjava/tjbot/commands/free/FreeChannelMonitor.java

```diff
@@ -25,14 +28,23 @@
  * After successful configuration, any changes in busy/free status will automatically be displayed
  * in the configured {@code Status Channel} for that guild.
  */
-final class ChannelMonitor {
+public final class FreeChannelMonitor {
```

Comment:
The monitor class is totally overloaded now. It was meant to only monitor channels (see its current Javadoc which u also forgot to adjust accordingly). But now its essentially the core business logic. It can not only monitor channels but also change status, build messages, send messages, find history, ...

I would advise to undo all changes on this class and introduce a `FreeSystem` or `FreeChannelSystem` class instead (similar to how `TagCommand` and similar use a `TagSystem` class), where you have all those utility methods. And then you would pass around the FreeSystem, not the monitor, to whoever is interested.

Also reads better in `Features` when you pass around a FooSystem rather than a monitor.